core/EventDispatcher.php

```diff
@@ -101,18 +101,22 @@ public function postEvent($eventName, $params, $pending = false, $plugins = null
 
         if (empty($plugins)) {
             $plugins = $manager->getPluginsLoadedAndActivated();
+        } else {
+            $pluginMap = [];
+            foreach ($plugins as $plugin) {
+                if (is_string($plugin)) {
+                    $plugin = $this->pluginManager->getLoadedPlugin($plugin);
```

Comment:
We have the manager in a variable and used elsewhere, so can use here as well
```suggestion
                    $plugin = $manager->getLoadedPlugin($plugin);
```